src/Codeception/Module/WebDriver.php

```diff
@@ -1676,7 +1685,7 @@ public function _loadSession($session)
      *
      * @api
      */
-    public function _backupSession()
+    public function _backupSession(): ?RemoteWebDriver
```

Comment:
@Naktibalda I just noticed that this method returns `: array` in Codeception 5
https://github.com/Codeception/Codeception/blob/a15b58781cbed2b3d5f21fd148cef474ee51101e/src/Codeception/Lib/Interfaces/MultiSession.php#L11
Should I remove the typing there?